client/Questions&Answers/AnswersList.jsx

```diff
@@ -27,17 +27,29 @@ function Answerslist() {
 
     <div className={expand}>
       {answers.map((answer) => (
-        <div key={answer.answer_id} className="answer">
+        <div key={answer.answer_id} className="view-answer">
           <div className="answerText">
             <strong>A:  </strong>
             {answer.body}
           </div>
-          <Test />
-          {
-            status === 'Report' ?
-              <SendFeedback option={0} handleFeedback={toggleStatus} />
-              : <SendFeedback option={1} handleFeedback={() => { console.log('This answer has already been reported.') }} />
-          }
+          <br />
+          <div className="answersFeedback">
+            <span className="answersFeedback-left">
+              by
+              {' '}
+              {answer.answerer_name}
+              {' '}
+              {answer.date}
+              {' '}
```

Comment:
I suspect these could be refactored to template literals. Something like

````
{`by ${answerer_name} ${answer.date} |`}
````

perhaps? Depends on what you're trying to display, of course, but I'll bet this could be condensed.